pilot/pkg/networking/core/v1alpha3/listener_builder.go

```diff
@@ -621,6 +622,9 @@ func (configgen *ConfigGeneratorImpl) buildInboundFilterchains(in *plugin.InputP
 			}
 		}
 	}
+	if in.Node.Type == model.SidecarProxy {
+		extension.AddWasmPluginsToMutableObjects(mutable, in.Push.WasmPlugins(in.Node))
```

Comment:
can you add some comment explaining why only sidecar?

Reply:
added comment

Reply:
@dgn i don’t see the comment. Did you push ?

Reply:
you'll have to the `Files changed` tab to see it, not sure why it doesn't show up here. see https://github.com/istio/istio/blob/ee35d0176ba4cfdc23099315a67e43755114d779/pilot/pkg/networking/core/v1alpha3/listener_builder.go#L625-L628

Reply:
Thanks @dgn. We need to make sure we have this behaviour specified in the API doc.